OpenGrok

```diff
@@ -995,8 +995,30 @@ DeployWar()
         FatalError "Missing Deployment Directory ${warTarget}"
     fi
 
+    APP_CONTEXT="source"
+    FINAL_WAR_NAME="source"
+    if [ "$applicationServer" = "Tomcat" ]
+    then
+        if [ ! -z ${OPENGROK_WEBAPP_CONTEXT+x} ]
+        then
+            APP_CONTEXT="${OPENGROK_WEBAPP_CONTEXT}"
+
+            # strip leading /
+            case ${APP_CONTEXT} in
+                /*) APP_CONTEXT=`echo ${APP_CONTEXT} | sed 's|^/||'`
+            esac
+
+            FINAL_WAR_NAME=`echo ${APP_CONTEXT} | sed 's|/|#|g'`
```

Comment:
how does this interact with the code below ?

```shell
    # OPTIONAL: Context URL of the OpenGrok webapp                              
    #           (default is /source)                                            
    WEBAPP_CONTEXT=""                                                           
    if [ -n "${OPENGROK_WEBAPP_CONTEXT}" ]; then                                
        WEBAPP_CONTEXT="-w ${OPENGROK_WEBAPP_CONTEXT}"                          
    fi
```

Reply:
What do you mean exactly? I used different variable names just to be safe. Also, the check
```bash
if [ -n "${OPENGROK_WEBAPP_CONTEXT}" ]; then 
```
does not seem to set the variable to empty value. Otherwise it would not work, right? Because this code is always executed.

Reply:
I was curious about the non-initial slash substitution. Should the parsing done in `Indexer#parseOptions()` be in line with the substitutions done in the script ? i.e. what happens if `OPENGROK_WEBAPP_CONTEXT` is set to `foo/bar` ?

Reply:
`Indexer#parseOptions()` does not seem to do much -> only adds `/` before and after the `OPENGROK_WEBAPP_CONTEXT` if needed and then uses the value like `${value}s?` for creating `href`s